src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,474 @@
+                          average_rain_intensity,
+                          event_duration,
+                          surface_type):
+    """
+    Calculates the total suspended solids (TSS) for a surface, given the following parameters;
+
+    Parameters
+    ----------
+    surface_area: float
+        surface area of the given surface type
+    antecedent_dry_days: float
+        length of antecedent dry period (days)
+    average_rain_intensity: float
+        average rainfall intensity of the event (mm/h)
+    event_duration: float
+        duration of the rainfall event (h)
+    surface_type: SurfaceType
+        the type of surface we are computing the TSS for
+
+    Returns
+    -------
+    float
+       Returns the TSS value from the given parameters
+    """
+
+    # Define the constants (Cf is the capacity factor).
+    roof_surface_types = {SurfaceType.CONCRETE_ROOF, SurfaceType.GALVANISED_ROOF, SurfaceType.COPPER_ROOF}
+    capacity_factor = 0.75 if surface_type in roof_surface_types else 0.25
+    # Values a1 to a3 are empirically derived coefficient values.
+
+    # Define error message (if needed)
+    invalid_surface_error = (f"Given surface is not valid for computing TSS."
+                             f" Needed a roof or road, but got {SurfaceType(surface_type).name}.")
+
+    a1, a2, a3 = 0, 0, 0
+    match surface_type:
+        case SurfaceType.CONCRETE_ROOF:
+            a1, a2, a3 = 0.6, 0.25, 0.00933
+        case SurfaceType.COPPER_ROOF:
+            a1, a2, a3 = 2.5, 0.95, 0.00933
+        case SurfaceType.GALVANISED_ROOF:
+            a1, a2, a3 = 0.6, 0.5, 0.00933
+        case SurfaceType.ASPHALT_ROAD | SurfaceType.CAR_PARK:
+            a1, a2, a3 = 2.9, 0.16, 0.0008
+        case _:
+            log.error(invalid_surface_error)
+            raise ValueError(invalid_surface_error)
+    first_term = surface_area * a1 * (antecedent_dry_days ** a2) * capacity_factor
+    second_term = (1 - math.exp(a3 * average_rain_intensity * event_duration))
+
+    return first_term * second_term
+
+
+def total_metal_load_roof(surface_area,
+                          antecedent_dry_days,
+                          average_rain_intensity,
+                          event_duration,
+                          rainfall_ph,
+                          surface_type):
+    """
+    Calculates the total metal load for a given roof;
+
+    Parameters
+    ----------
+    surface_area: float
+        surface area of the given surface type
+    antecedent_dry_days: float
+        length of antecedent dry period (days)
+    average_rain_intensity: float
+        average rainfall intensity of the event (mm/h)
+    event_duration: float
+        duration of the rainfall event (h)
+    rainfall_ph: float
+        the acidity level of the rainfall
+    surface_type: int
+        the type of roof we are calculating the metal load for. Some coefficients depend on this.
+
+    Returns
+    -------
+    (float, float)

```

Comment:
Go though the whole PR and change each of these, to match te rest of our codebase and help with IDE and autodoc support
```suggestion
    tuple[float, float]
```